source/DEVS-Network/DEVSNetwork.class.st

```diff
@@ -0,0 +1,28 @@
+Class {
+	#name : 'DEVSNetwork',
+	#superclass : 'DEVSCoupledComponent',
+	#category : 'DEVS-Network',
+	#package : 'DEVS-Network'
+}
+
+{ #category : 'modeling' }
+DEVSNetwork >> initialize [ 
+	| serverName router host |
+	super initialize.
+	
+	router := DEVSNetworkRouter new name: #router.
+	self addComponent: router.
+	1 to: 3 do: [ :n |
+		serverName := ('server', n asString).
+		host := DEVSNetworkHost new
+			name:  ('host:10.0.0.', n asString);
+			hostname: 'server', n asString, '.local';
+			address: ('10.0.0.', n asString).
+		self addComponent: host.
+		router addHost: host.
+		
+		self addCouplings: { 
+			{ host name . #out } -> { router name . #in}.
+			{ router name . host address } -> { host name . #in } }.
+		]
```

Comment:
This is potentially a piece of code that should be refactored to own builder right? 
So I would imagine that is should look like this:
```
DEVSNetwork new
   name: 'UDP network simulation';
   buildHostsInRange: hostNamesAndIPAddresses;
   inPortsRange: IPAddressesAndPorts;
   yourself
```
In theory, current hardcoded hosts could remain, if nothing else is happened. If `buildHostsInRange: ` is sent, then existing hosts with couplings will be removed and added new as specified in hostnamesAndIPAddresses parameter. Similar to ìnPortsRange:` where network model would delegate to hosts with desired IP address.
But this is just idea for future improvement (not that something blocking for this PR).

Reply:
Yes this code will be refactored